src/duration.rs

```diff
@@ -347,12 +347,21 @@ impl Duration {
     ///
     /// This function errors when duration is less than zero. As standard
     /// library implementation is limited to non-negative values.
+    // TODO: make this method const once our MSRV is 1.58+
     pub fn to_std(&self) -> Result<StdDuration, OutOfRangeError> {
         if self.secs < 0 {
             return Err(OutOfRangeError(()));
         }
         Ok(StdDuration::new(self.secs as u64, self.nanos as u32))
     }
+
+    /// Returns `true` if d < MIN || d > MAX
+    const fn out_of_bounds(&self) -> bool {
```

Comment:
This should be a constructor, not a method on an already constructed type.

Reply:
You are completely right!

Actually I wanted to have a `Duration::new` in https://github.com/chronotope/chrono/pull/1310 and https://github.com/chronotope/chrono/pull/1290.